### PR TITLE
Added Ath wiki to docs branch

### PR DESCRIPTION
**Pull request by weihao**
### Type
- [ ] Internal change (Doesn't affect end-user).
- [ ] External change (Does affect end-user).
- [x] Wiki (Changes towards the [Wiki]).
- [ ] Other: __________ 

### Description
Added placeholders for `Ath` plugin.

Closes N/A 

[Wiki]: https://github.com/PlaceholderAPI/PlaceholderAPI/wiki

### CODE REVIEW

File: wiki/Home.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ It also has a list with all available placeholders (Work in progress).
   - [[Animations|Placeholders#animations]]
   - [[Armor|Placeholders#armor]]
   - [[ASCII|Placeholders#ASCII]]
+  - [[Ath|Placeholders#ath]]
```
Comment (Andre601) [inline wiki/Home.md:27]: Plugin expansions aren't listed here. Please remove.

File: wiki/Placeholders.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ If the command itself isn't there and `NO DOWNLOAD COMMAND` instead is shown, th
 - **[Animations](#animations)**
 - **[Armor](#armor)**
 - **[ASCII](#ascii)**
+- **[Ath](#ath)**
```
Comment (Andre601) [inline wiki/Placeholders.md:18]: Plugin expansions aren't listed here. Please remove.

File: wiki/Placeholders.md [hunk 3]
```diff
@@ -1238,6 +1240,15 @@ All information about these placeholders can be found [here](https://gitlab.com/
 ```
 ----
 
+- ### **[Ath](https://www.spigotmc.org/resources/ath-peak-players-record-spigot-and-bungeecord-support.87124/)**
```
Comment (Andre601) [inline wiki/Placeholders.md:1243]:
```suggestion
- ### **[Ath](https://www.spigotmc.org/resources/87124/)**
```

File: wiki/Plugins-using-PlaceholderAPI.md [hunk 4]
```diff
@@ -89,6 +89,9 @@ If your plugin isn't shown here and you want it to be added, [open an issue](/Pl
 - **[AsyncKeepAlive](https://www.spigotmc.org/resources/64676/)**
   - [ ] Supports placeholders.
   - [x] Provides own placeholders. [**[[Link|Placeholders#asynckeepalive]]**]
+- **[Ath](https://www.spigotmc.org/resources/ath-peak-players-record-spigot-and-bungeecord-support.87124/)**
```
Comment (Andre601) [inline wiki/Plugins-using-PlaceholderAPI.md:92]:
```suggestion
- **[Ath](https://www.spigotmc.org/resources/87124/)**
```

File: wiki/_Sidebar.md [hunk 5]
```diff
@@ -24,6 +24,7 @@
   - [[Animations|Placeholders#animations]]
   - [[Armor|Placeholders#armor]]
   - [[ASCII|Placeholders#ASCII]]
+  - [[Ath|Placeholders#ath]]
```
Comment (Andre601) [inline wiki/_Sidebar.md:27]: Plugin expansions aren't listed here. Please remove.

**Review by Andre601 — CHANGES_REQUESTED** [review]
Looks alright but has some mistakes in it that needs correction

File: wiki/Placeholders.md (hunk 2 above)
Comment (weihao) [reply wiki/Placeholders.md]: Removed.

File: wiki/Home.md (hunk 1 above)
Comment (weihao) [reply wiki/Home.md]: Removed.

**Review by Andre601 — APPROVED** [review]

**Pull request merged**